CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ else()
 
   ##############################################################################
   # - Prepare rapids-cmake -----------------------------------------------------
-  file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-21.12/RAPIDS.cmake
+  file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.04/RAPIDS.cmake
```

Comment:
I wonder if we should pin a particular tag for rap-cmake

Reply:
The one for tomorrow or Friday? Going by your comment above for specific branches